ui/components/app/name/name.tsx

```diff
@@ -101,7 +101,7 @@ export default function Name({
         onClick={handleClick}
       >
         {hasDisplayName ? (
-          <Identicon address={value} diameter={16} />
+          <Identicon address={value} diameter={16} image={image} />
```

Comment:
To clarify, does a provided `image` always take priority?

Reply:
Yes it does https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/ui/identicon/identicon.component.js#L185